src/App.js

```diff
@@ -1,23 +1,26 @@
 import logo from './logo.svg';
+import {Routes, Route} from 'react-router-dom';
+import { useState, useEffect } from 'react'
```

Comment:
Here you didn't need to import useState, useEffect, axios since you're not using them